### ISSUE TITLE
chance for..  looting pokéstops, too? (more about my idea, in descript) 

### ISSUE THREAD

**Issue by anyDroidzOS** [pokesniperandroid/PokeSniper-Android#206]
is it available, to make a feature, thats let us loot pokéstops, in home coordinates? 

(..or gives a way 4 u, to make 2nd app/tool, but for looting pokéstops only ?, may w/ itembag on sidemenue or anything what are makeable)?

**Comment by Av1s** [comment]
yeah @pokesniperandroid it would be a nice addition to the app if you could loot a pokestop at home coords if there r any.... 

if u made this happen i for sure will donate some bucks for it😊 keep up the good work

**Comment by DrakeLyon** [comment]
I need this feature. I'm not able to go out because my wife doesn't like me going anywhere without her and she doesn't play Pokémon GO lol. If we could have this app just search any Pokéstops at certain coordinates when we run out of Pokéballs it would help out a lot
